arcade/examples/sound_demo.py

```diff
@@ -1,45 +1,50 @@
 """
-Starting Template
+Sound Panning Demo
 
-Once you have learned how to use classes, you can begin your program with this
-template.
-
-If Python and Arcade are installed, this example can be run from the command line with:
 python -m arcade.examples.sound_demo
 
 The top button is to play a music track.
-The 3 rows of buttons are arranged such that the audio is panned in the direction of the button,
-and the volume increases as you go down the column.
+The 3 rows of buttons are arranged such that the audio is panned in the
+direction of the button, and the volume increases as you go down the column.
 
 Left click a button to play a sound. 
 If a sound is playing right click to increase volume, middle click to decrease.
 """
+
 import typing
 
 import arcade
 
 SCREEN_WIDTH = 800
 SCREEN_HEIGHT = 600
-SCREEN_TITLE = "Starting Template"
+SCREEN_TITLE = "Sound Panning Demo"
 BUTTON_SIZE = 30
 
+SOUND_PANNING = [-1.0, -0.5, 0.0, 0.5, 1.0]
+assert len(SOUND_PANNING) == 5
```

Comment:
This might not make sense to keep. It would be better to move related constants here with a comment, or dynamically generate this.